content/en/data_jobs/databricks.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Enable Data Jobs Monitoring for Databricks
+kind: documentation
+further_reading:
+    - link: '/data_jobs'
+      tag: 'Documentation'
+      text: 'Data Jobs Monitoring'
+---
+
+{{< callout url="https://forms.gle/PZUoEgtBsH6qM62MA" >}}
+Data Jobs Monitoring is in private beta. Fill out this form to join the wait list.
+{{< /callout >}} 
+
+[Data Jobs Monitoring][7] gives visibility into the performance and reliability of your Apache Spark and Databricks jobs.
+
+## Setup
+
+Follow these steps to enable Data Jobs Monitoring for Databricks.
+
+1. [Configure the Datadog-Databricks integration](#configure-the-datadog-databricks-integration) with your Databricks API token.
+1. [Install the Datadog Agent](#install-the-datadog-agent-on-your-databricks-clusters) on your Databricks cluster(s).
+1. [Add your Datadog API key](#add-your-datadog-api-key-in-databricks) in Databricks.
+
+### Configure the Datadog-Databricks integration 
+
+1. In your Databricks workspace, click **User Settings**. Then click on **Access Tokens**.
```

Comment:
Databricks recently slightly changed their UI

The new path for Access tokens is: User Settings -> Developer Tab -> Manage Access tokens

<img width="1292" alt="image" src="https://github.com/DataDog/documentation/assets/99249445/e955e1cf-dc67-48c2-bf5a-150af1ce77bb">